infra/build_deploy_config.py

```diff
@@ -89,8 +110,16 @@
     "GCP_DEFAULT_REGION",
 ]
 
+if not validate_aws_config():
+    exit(1)
+
+caller_info = boto3.client("sts").get_caller_identity()
```

Comment:
`boto3.client("sts").get_caller_identity()` and `caller_info['UserId'].split(':')[1]` are already declared above and could be reused in `validate_aws_config()` if they're going to be naked vars.

Reply:
thanks, ill refactor